src/components/image-input/image-input.react.js

```diff
@@ -59,6 +64,9 @@ export default class ImageInput extends PureComponent<TProps> {
           tabIndex="0"
         />
         <div className={styles['ui-image-input__pseudo']}>{placeholder}</div>
+        <div className="ui-image-input__recommendation">
+          <p>{recommendation}</p>
```

Comment:
I have a feeling that the root `<p>` tags get slightly different styling in the frontend versus identity/pay etc.

Could use the `Text` component, or add some component specific classes/styles to the tag?